### PR TITLE
chore: upgrade parachain and clients to v1.0.x

### PR DESCRIPTION
**Pull request by gregdhill**
Signed-off-by: Gregory Hill <gregorydhill@outlook.com>

### CODE REVIEW

File: docs/vault/installation.md [hunk 1]
```diff
@@ -256,7 +256,8 @@ RUST_LOG=info cargo run -p vault -- \
   --auto-register-with-faucet-url 'https://api.interlay.io/faucet' \
   --telemetry-url 'https://api.interlay.io/telemetry' \
   --btc-parachain-url 'wss://api.interlay.io/parachain' \
-  --network=testnet
+  --network=testnet \
+  --currency-id=dot
```
Comment (nud3l) [inline docs/vault/installation.md:260]: I think this will not work. Should be `--currency-id DOT`?

File: scripts/vault/interbtc-vault.service [hunk 2]
```diff
@@ -14,7 +14,8 @@ ExecStart=/opt/interbtc/vault/vault \
   --auto-register-with-faucet-url 'https://api.interlay.io/faucet' \
   --telemetry-url 'https://api.interlay.io/telemetry' \
   --btc-parachain-url 'wss://api.interlay.io/parachain' \
-  --network=testnet
+  --network=testnet \
+  --currency-id=dot
```
Comment (nud3l) [inline scripts/vault/interbtc-vault.service:18]: similar to above

File: docs/vault/installation.md (hunk 1 above)
Comment (gregdhill) [reply docs/vault/installation.md:260]: No I think we take the lowercase of the value anyway

Comment (nud3l) [reply docs/vault/installation.md:260]: But the `=` did not work for me locally at least

**Review by nud3l — APPROVED** [review]

**Pull request merged**